### PR TITLE
CUTEst_jll 2.0.5: add missing script

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: C/CUTEst/build_tarballs.jl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder
 
 name = "CUTEst"
-version = v"2.0.4"
+version = v"2.0.5"
```
Comment (giordano) [inline C/CUTEst/build_tarballs.jl:6]: You didn't change the sources?

Comment (dpo) [reply C/CUTEst/build_tarballs.jl:6]: Just the build_tarballs, which will affect the jll. Should I not modify the version number?

Comment (giordano) [reply C/CUTEst/build_tarballs.jl:6]: https://juliapackaging.github.io/BinaryBuilder.jl/dev/building/#Version-number

Comment (dpo) [reply C/CUTEst/build_tarballs.jl]: Thanks.

Comment (giordano) [reply C/CUTEst/build_tarballs.jl]: We're probably going to change the versioning scheme soon, but for the time being better to keep consistent

**Pull request merged**